eng/Versions.props

```diff
@@ -3,7 +3,7 @@
     <VersionPrefix>3.3.1</VersionPrefix>
     <PreReleaseVersionLabel>beta1</PreReleaseVersionLabel>
     <NetAnalyzersVersionPrefix>5.0.0</NetAnalyzersVersionPrefix>
-    <NetAnalyzersPreReleaseVersionLabel>rc2</NetAnalyzersPreReleaseVersionLabel>
+    <NetAnalyzersPreReleaseVersionLabel>GA</NetAnalyzersPreReleaseVersionLabel>
```

Comment:
Do we want to create a separate branch for GA?

Reply:
changed this to merge into `release/5.0.1xx`